README.md

```diff
@@ -1 +1,46 @@
-# Capstone1
\ No newline at end of file
+# Microverse HTML, CSS, JS Capstone Project
+"This website is an announcement for the upcoming Launch event. The event will be held within Yıldız Technical University Technology Club. General information about the event is given throughout the site. Speakers are introduced in a separate section."
+
+The website content is fictional. It was created using HTML, CSS and JavaScript to meet the requirements defined within the scope of the Microverse Capstone project. In particular, I would like to point out that I built this website completely from scratch.
+
+
+![lumato](https://user-images.githubusercontent.com/75973193/125121373-80aaf500-e0fc-11eb-9f5b-442ea041ce20.png)
+
+
+
+It is built as a part of Microverse Web Developer Course. The mobile-first approach was used in this project.
+
+## Built With
+
+-HTML
+-CSS
+-Javascript
+
+## Project Description
+
+The project you are going to build is based on an *online website for launch event*. We provide some design guidelines for you to create the website. It's very important the you personalize your project so that you have something unique in your portfolio to share with potential employers during job searching.
+
+## Getting Started
+
+To get a local copy up and running usual steps.
+
+
+## Author
+👤 **Tacettin Kustu**
+
+- GitHub: [@isaka-lumato](https://github.com/isaka-lumato)
+- LinkedIn: [tacettinkustu](https://www.linkedin.com/in/isaka-william-90773020b/)
```

Comment:
- [ ] Kindly check on your social media handles and ensure that the handles have the same names